src/js/view/controls/controls.js

```diff
@@ -102,7 +103,7 @@ export default class Controls extends Events {
 
         const touchMode = model.get('touchMode');
 
-        const focusPlayerElement = () => {
+        this.focusPlayerElement = () => {
```

Comment:
need this function for `userInactive` below but the reliance on the `model` means we need to define it here or refactor more